webpack/JobWizard/JobWizard.js

```diff
@@ -9,23 +9,35 @@ import { AdvancedFields } from './steps/AdvancedFields/AdvancedFields';
 import { JOB_TEMPLATE } from './JobWizardConstants';
 import { selectTemplateError } from './JobWizardSelectors';
 import Schedule from './steps/Schedule/';
+import HostsAndInputs from './steps/HostsAndInputs/';
 import './JobWizard.scss';
 
 export const JobWizard = () => {
   const [jobTemplateID, setJobTemplateID] = useState(null);
   const [category, setCategory] = useState('');
+
   const [advancedValues, setAdvancedValues] = useState({});
+  const [templateValues, setTemplateValues] = useState({}); // TODO use templateValues in advanced fields - description
+  const [selectedHosts, setSelectedHosts] = useState(['host1', 'host2']);
```

Comment:
can we use actual data here?

Reply:
what should be the flow? selecting `Hosts` will select all hosts that are available for the user? or can we select just specific hosts? the same question goes for the other select options.

is there another PR/Redmine that handles that?

Reply:
https://github.com/theforeman/foreman_remote_execution/pull/578